Description: Allow guest session processes to read /proc/<PID>/stat
Author: Tyler Hicks <tyhicks@canonical.com>

Index: lightdm-1.9.13/data/apparmor/abstractions/lightdm
===================================================================
--- lightdm-1.9.13.orig/data/apparmor/abstractions/lightdm	2014-04-02 16:43:27.946041262 -0500
+++ lightdm-1.9.13/data/apparmor/abstractions/lightdm	2014-04-02 16:44:54.350039489 -0500
@@ -47,6 +47,8 @@
   owner @{PROC}/** rm,
   # needed for gnome-keyring-daemon
   @{PROC}/*/status r,
+  # needed for bamfdaemon and utilities such as ps and killall
+  @{PROC}/*/stat r,
   /sbin/ r,
   /sbin/** rmixk,
   /sys/ r,
